_posts/2016-05-11-android-testing-with-kotlin.md

```diff
@@ -0,0 +1,422 @@
+---
+layout: post
+title:  "Using Kotlin to test Android applications"
+date:   2016-05-11 00:00:00
+categories: Greenhouse update
+tags: [android, testing, kotlin, java]
+author: priit
+share: true
+published: true
+comments: true
+---
+
+Testing is an integral part of any kind of software development.
+To ensure that code is ready to be deployed to production it is rarely
+enough to just manually verify that new features are working.
+When talking about code quality and reliability, importance of automated
+testing cannot be overstated, be it regression tests, unit tests or UI tests.
+
+In this blog post we discuss yet another possibility of how to write automated
+tests for your Android applications. Namely, we observe how [**Kotlin**](https://kotlinlang.org/),
+one of the latest stable and noteworthy additions to the vast world of JVM based
+languages, can be utilized for test automation of Android applications.
+
+<!--more-->
+
+## Few words about Kotlin
+
+Kotlin is an [open source language](https://github.com/JetBrains/kotlin)
+developed and maintained by JetBrains, the company behind IntelliJ IDEA and
+possibly your favorite IDE for Android development, Android Studio.
+
+After being in development for more than 5 years (first commit dating back to 2010-11-08),
+the first stable version of the language [was released](http://blog.jetbrains.com/kotlin/2016/02/kotlin-1-0-released-pragmatic-language-for-jvm-and-android/)
+earlier this year in February.
```

Comment:
"earlier this year in February" is a bit redundant. Better to keep either "earlier this year" or "in February of this year"